client/my-sites/plans/README.md

```diff
@@ -18,3 +18,16 @@ React component that renders a list of active WordPress.com plans
 
 ### plans-compare.jsx
 React component that renders a comparison table of active WordPress.com plans
+
+## Initializing checkout flow
+
+To start a plan checkout flow, use appropriate URLS:
+
+- /checkout/site.wordpress.com/premium
+- /checkout/site.wordpress.com/personal
+- /checkout/site.wordpress.com/business
+- /checkout/site.jetpack.me/premium
+- /checkout/site.jetpack.me/professional
```

Comment:
We'll also need a jetpack `premium-monthly` and `professional-monthly`. Though feel free to discuss the naming of those slugs